src/commands/listIsActive.ts

```diff
@@ -1,10 +1,13 @@
 import { EditorState } from "prosemirror-state"
 import { NodeType, Schema } from "prosemirror-model"
-import { hasParentNodeOfType } from "prosemirror-utils"
+import { findParentNodeOfType } from "prosemirror-utils"
 
 export function listIsActive(
   state: EditorState,
-  type: NodeType<Schema<any, any>>
+  type: NodeType<Schema<any, any>>,
+  listNodes: typeof type[]
 ) {
-  return hasParentNodeOfType(state.schema.nodes[type.name])(state.selection)
+  const listNode = findParentNodeOfType(listNodes)(state.selection)
```

Comment:
I think we can use `findParentNodeOfTypeClosestToPos` method from [prosemirror-utils](https://github.com/atlassian/prosemirror-utils)

Reply:
Oh that's a handy util 👍 looks good to me